policy-service/src/main/java/uk/gov/gchq/palisade/service/policy/config/StdPolicyPrepopulationFactory.java

```diff
@@ -71,6 +71,7 @@ public StdPolicyPrepopulationFactory(final String resourceId, final Map<String,
         this.recordRules = recordRules;
     }
 
+    @SuppressWarnings({"java:S3740", "rawtypes"}) //Suppress parameterized type for generic class smell
```

Comment:
Having trouble getting the comment to be on line 66, but the suggestion is to change "data-service" to Data Service.